Tamil/Thirukural.md

```diff
@@ -8,4 +8,8 @@ Friendship hastens to rescue as fast as hand’s reflex to catch slipping cloth
 தெய்வத்தான் ஆகா தெனினும் முயற்சிதன்
 மெய்வருத்தக் கூலி தரும்
 Though fate-divine should make your labour vain;
-Effort its labour's sure reward will gain
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.